flowpipe/graph.py

```diff
@@ -259,134 +259,39 @@ def evaluate(self, mode="linear", skip_clean=False,
             data_persistence (bool): If false, the data on plugs that have
                 connections gets cleared (set to None). This reduces the
                 reference count of objects.
+            evaluator (flowpipe.evaluators.Evaluator): The evaluator to use.
+                For the basic evaluation modes will be picked by 'mode'.
```

Comment:
Maybe we should deprecate this method? Adding a print on how to use the Evaluator instead

Reply:
I think having a `Graph.evaluate()` method will be very useful for new users. Flowpipe already has a lot of uncommon concepts flying around. If a user wants a simple evaluation scheme to be applied, I'd prefer to provide one with a lot of ease of use.

I could move the `data_persistence` stuff to the evaluator though, and drop handing in a custom evaluator to this method. Then one could use "vanilla" evaluators out of the box, or set up and use a custom evaluator.

Reply:
Good point, let's do it like that!